memory_profiler.py

```diff
@@ -808,6 +809,25 @@ def inner_wrapper(f):
             return profile(f, stream=stream, precision=precision)
         return inner_wrapper
 
+class LogFile(object):
+    """File-like object to log text using the `logging` module and the log report can be customised."""
```

Comment:
Could you please describe in the docstring the arguments name and reportIncrementFlag ?